src/main/java/Duke.java

```diff
@@ -1,10 +1,175 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>(100);
+        String message = "____________________________________________________________\n" +
+                "Hello! I'm Jux\n" +
+                "What can I do for you?\n";
+        String end = "____________________________________________________________\n" +
+                "Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________"
+                ;
+        FileManipulation dataFile = new FileManipulation();
+        if (dataFile.getDoesExist()) {
+            try {
+                dataFile.loadFile(list);
+                for (int i = 0; i < list.size();i++) {
+                    System.out.println(list.get(i));
+                }
+
+            } catch(FileNotFoundException e) {
+                System.out.println("File not found");
+            }
+        } else {
+            System.out.println(message);
+        }
+
+        while(true) {
+
+            try {
+                String input = scanner.nextLine();
+                if (input.equals("bye")) {
+                    break;
+                } else if(input.equals("list")) {
+                    if (list.isEmpty()) {
+                        throw new DukeException(" YOUR LIST IS EMPTY");
+                    }
+                    System.out.println("Here are the tasks in your list!");
+                    for (int i = 0; i < list.size();i++) {
+                        int j = i + 1;
+                        String listMessage = j + "." +  list.get(i);
+                        System.out.println(listMessage);
+                    }
+                } else if(input.startsWith("mark")) {
+
+                    if (input.length() > 5 ) {
+                        String listStringNumber =  input.substring(5);
+                        // future error detection for non-numerals
+                        int convertedToNumber = Integer.parseInt(listStringNumber) - 1;
+                        // future error when list is empty
+                        if (convertedToNumber < 0 || convertedToNumber >= list.size()) {
+                            throw new DukeException("NUMBER NOT IN LIST, PLEASE ADD A TASK OR " +
+                                    "CHOOSE A DIFFERENT NUMBER WITHIN 1 AND"
+                                    + list.size());
```

Comment:
I like how you provide users with thorough instructions on how to enter the right data!